### DISCUSSION TITLE
Frontend menu editor

### DISCUSSION THREAD

**Discussion by MDylan** [area17/twill#2079] [Q&A]
Hi!

Is there any suggested / ready solution to how to manage frontend menu with twill? 

Thank you!

**Comment by haringsrob** [comment]
hey @MDylan,

In twill 3 there is an example that has a menu: https://github.com/area17/twill/tree/3.x/examples/basic-page-builder

**Reply by MDylan** [reply]
Ok thanks I tried it (btw it's not ready for use now, some bug in there), but is only one menu. 
If I have multiple menu on the frontend, I need to create more model for this.

**Reply by haringsrob** [reply]
Hey @MDylan there are other ways for this for example:

A menu is a "menu" entity, which has parent-child with MenuItems. 

Where menuitems is similar to what I linked earlier.

However if you have 2, 3 fixed menu's, separate models might be more interesting as you do not need to handle where a dynamic menu is placed.

https://twill.io/docs/crud-modules/nested-modules.html
